sdk/attestation/azure-security-attestation/src/test/java/com/azure/security/attestation/AttestationClientTestBase.java

```diff
@@ -60,16 +71,44 @@
 /**
  * Specialization of the TestBase class for the attestation tests.
  * <p>
- * Provides convenience methods for retrieving attestation client builders, verifying attestation tokens,
- * and accessing test environments.
+ * Provides convenience methods for retrieving attestation client builders, verifying attestation tokens, and accessing
+ * test environments.
  */
-@Execution(ExecutionMode.SAME_THREAD)
 @Isolated
-public class AttestationClientTestBase extends TestBase {
-
-    protected final ClientLogger logger = new ClientLogger(AttestationClientTestBase.class);
+public class AttestationClientTestBase extends TestProxyTestBase {
+    protected static final SerializerAdapter ADAPTER = JacksonAdapter.createDefaultSerializerAdapter();
+    protected static final String DISPLAY_NAME_WITH_ARGUMENTS = "{displayName} with [{arguments}]";
+    protected static final ClientLogger LOGGER = new ClientLogger(AttestationClientTestBase.class);
     protected Tracer tracer;
 
+    @Override
+    protected void beforeTest() {
+        super.beforeTest();
+
+        if (interceptorManager.isPlaybackMode()) {
+            interceptorManager.addMatchers(Collections.singletonList(new CustomMatcher()
+                .setHeadersKeyOnlyMatch(Collections.singletonList("Authorization"))));
+        }
+    }
+
+    private static final TestMode TEST_MODE = getTestModeLocal();
+
+    static TestMode getTestModeLocal() {
```

Comment:
why do we need this?

Reply:
There's nowhere in azure-core-test that exposes the static TestMode value. We should think about exposing that publicly next release.